### DISCUSSION TITLE
OTA - What binary to choose from?

### DISCUSSION THREAD

**Discussion by djiwondee** [tbnobody/OpenDTU#1367] [Q&A]
Hello Community!

I've a working setup of OpenDTU, which has been implemented  by flashing the firmware to an ESP32 MCU using VSCode. Now I would like update the firmware OTA. I did already a research in the WIKI of this repo and within this forum as well. I haven't found an clear answer so far to what I have looked for:

- Can I perform and OTA, even haven't applied a precompiled binary before?
- If yes, which is the right binary I have to choose from the [Asset page](https://github.com/tbnobody/OpenDTU/releases)?

I would assume `opendtu-generic_esp32.bin`. Am I right? I'm especially confused whether the file with or without `.factory` is the right one.

Thanks in advance!

**Comment by DPO99** [comment]
Could work. I would try "opendtu-generic.bin" (needs no profile = standard config!) but i think you need to flash it completely new.

.factory is to flash it completely new.
